pilot/pkg/xds/sds.go

```diff
@@ -154,7 +154,7 @@ func (s *SecretGen) Generate(proxy *model.Proxy, push *model.PushContext, w *mod
 			secret, err := secretController.GetCaCert(sr.Name, sr.Namespace)
 			if err != nil {
 				pilotSDSCertificateErrors.Increment()
-				log.Warnf("failed to fetch ca certificate for %v: %v", sr.ResourceName, err)
+				log.Warnf("failed to fetch ca certificate for %s: %v", sr.ResourceName, err)
```

Comment:
I don't mind this but don't think its important to refactor around, as it will just add more noise with little benefit?